grails-app/domain/org/pih/warehouse/shipping/ShipmentType.groovy

```diff
@@ -37,4 +42,10 @@ class ShipmentType implements java.io.Serializable {
 
     String toString() { name }
 
+    String getDisplayName() {
+        def g = ApplicationHolder.application.mainContext.getBean( 'org.codehaus.groovy.grails.plugins.web.taglib.ApplicationTagLib' )
+        String defaultName = LocalizationUtil.getDefaultString(name)
+        return g.message(code: "react.stockMovement.shipmentType.${defaultName.toLowerCase()}.label", default: name)
+    }
+
```

Comment:
this handles the translations of shipment types that crowdin will "understand". There has been a ticket for that in the backlog that has been there for a long time, because we didn't have an idea how to handle that, since those were translated in an old way using `:fr` separators.

Reply:
Just to be completely accurate, I think you mean the pipe-delimited convention (e.g. sometext|fr:somefrenchtext).

Reply:
Yeah, exactly, I apologize for the shorthand